.github/workflows/stale.yml

```diff
@@ -0,0 +1,47 @@
+name: Mark stale issues and pull requests
+
+on:
+  schedule:
+    - cron: "0 12 * * 0"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      # 120+30 day stale policy for PRs
+      # * Except PRs marked as "no stale"
+      - name: Stale PRs policy
+        uses: actions/stale@v4
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          exempt-pr-labels: "no stale"
+          days-before-stale: 120
+          days-before-close: 30
+          days-before-issue-stale: -1
+          days-before-issue-close: -1
+          remove-stale-when-updated: true
+          stale-pr-label: "stale"
+          stale-pr-message: >
+            There hasn't been any activity on this pull request recently. It has been
+            automatically marked as stale because of that and it will be closed if no
+            further activity occurs in the next 30 days.
+
+      # 60+30 day stale policy for issues pending more information
+      # * Issues that are pending more information
+      # * Except Issues marked as "no stale"
+      - name: Needs more information stale issues policy
+        uses: actions/stale@v4
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          only-labels: "needs more information"
```

Comment:
Not all issues will be marked with this label. For example, old ones that maybe should stale don't have it, so they will remain as they are now... Is that the desired behavior? Maybe just having `no stale` to explicitly avoid it is enough?

Reply:
Indeed, even further this label is not one of the default ones in our repos.. so I will probably never be used.

However I'm not sure what we want here..

Issues are not like PRs, where if the author stops taking care of it we want to close & dismiss. IMO, we only want to close issues if:

- They have been fixed
- They're not really issues (not in the scope of the module, etc)
- They're not clear / unable to reproduce

But if an issue is valid and remains open, no matter for how long, we shouldn't close it I think.

---

On the other hand.. keeping a long list of probably outdated and unqualified issues is not helping newcomers or people eager to contribute to take care of them :sweat_smile:  so maybe it isn't such a bad idea to have them stale unless we qualify them with "no stale"

So you see, I'm not sure what's best here. Opinions?

Reply:
I prefer to close them. Issues for old versions don't add too much value. If the error is still reproducible (and applicable) to newer versions, it can be re-opened retargeting them or create new ones.

Reply:
How about now?

I kept two rules for issues:

1) Applies on all issues: `180+30 day stale`

2) Applies on issues with "needs more information": `60+30 day stale policy`


Reasoning behind rule 2 is that we want to give some time for the author to clearify the issue (so we tag as needs more information), and if he/she doesn't reply in 60 days, we stale it automatically (then close in another 30 days)

Reply:
Ok, that seems like a good compromise :+1: 

About the code, will these 2 rules applying to issues conflict? It looks like they won't, AFAICS, but just to check... You could probably add `needs more information` in the `exempt-issue-labels` key for the `180+30` policy to ensure they won't conflict...

Reply:
About https://github.com/OCA/oca-addons-repo-template/pull/63#discussion_r676637246: I agree with @joao-p-marques and @pedrobaeza. If some issue is valid and yet old, we just have to add the label.

Here in OCA opening issues is not as helpful as in other communities. Usually having an issue is good because the code grows and it should be fixed. However here our code doesn't grow into one direction, but in many (one per version), so as @pedrobaeza said an issue for an older Odoo version can be more a burden than a help.

Also here maintainers only maintain the parts of code they care about. One day a module may stop being used and their issues won't be fixed, so it's better let the module die along with its issues and PRs.

And if some issue is persistent, it's just a matter of replying to the bot once every 3 months. Fair enough.

---

About https://github.com/OCA/oca-addons-repo-template/pull/63#discussion_r676847122, it seems OK to me to stale incomplete issues sooner. Actually I think maybe 60+30 days is too much. I'd say 15+30. I think that, if 2 months later some bot reminds you to give more details about an issue, you just won't remember them. 2 weeks seems better IMHO. OK to that anyway.

Reply:
I've just pushed https://github.com/OCA/oca-addons-repo-template/pull/63/commits/f6b3ca5f7c933ea923f466dcd978195048dfb92b attending to both @Yajo and @joao-p-marques suggestions :)